source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -214,7 +214,7 @@ Follow the steps below to use the `kid` to determine which public key from the D
   * Integration: [https://identity.integration.account.gov.uk/.well-known/did.json](https://identity.integration.account.gov.uk/.well-known/did.json) 
   * Production: [https://identity.account.gov.uk/.well-known/did.json](https://identity.account.gov.uk/.well-known/did.json). 
 1. Make sure the controller ID matches the `id` in the DID document.
-1. Find the object in `assertionMethods` which has an `id` field matching the `kid` from the JWT header. If there are multiple keys in the DID document, GOV.UK One Login is in the process of rotating its keys. If there’s a key without a matching `id`, do not trust the identity and contact GOV.UK One Login to report an incident.
+1. Find the object in `assertionMethods` which has an `id` field matching the `kid` from the JWT header. If there are multiple keys in the DID document, GOV.UK One Login is in the process of rotating its keys. If there’s a `kid` without a matching `id`, do not trust the identity and contact GOV.UK One Login to report an incident.
```

Comment:
```suggestion
1. Find the object in `assertionMethods` which has an `id` field matching the `kid` from the JWT header. If there are multiple keys in the DID document, GOV.UK One Login is in the process of rotating its keys. If you have received a `kid` that does not correspond to an `id` in the DID document, do not trust the identity and contact GOV.UK One Login to report an incident.
```

Reply:
Might be one for the tech writers to give a quick opinion on?

Reply:
I wonder if it should be in present tense?
```
If you receive a `kid` that does not correspond to an `id` in the DID document
```